soc/aesc/vexriscv/Kconfig.soc

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2025 aesc silicon
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_AESC_VEXRISCV
```

Comment:
I'm a bit confused by the whole `aesc`, `vexrisc`, `nitrogen`, `elemrv`, `n` naming here. Is the family really vexriscv?

Reply:
Actually after checking this comment I figured out that `SOC_FAMILY_AESC_VEXRISCV` is wrong and not required at all. I removed all these configs here.